tests/end_to_end_test/snapshot_actions_test.go

```diff
@@ -219,6 +220,50 @@ func TestSnapshotActionsBeforeAfterFolder(t *testing.T) {
 	e.RunAndExpectFailure(t, "snapshot", "create", rootDir)
 }
 
+func TestSnapshotActionsEmbeddedScript(t *testing.T) {
+	t.Parallel()
+
+	e := testenv.NewCLITest(t)
+
+	e.RunAndExpectSuccess(t, "repo", "create", "filesystem", "--path", e.RepoDir)
+	defer e.RunAndExpectSuccess(t, "repo", "disconnect")
+
+	var (
+		successScript      = tmpfileWithContents(t, "echo Hello world!")
+		successScript2     = tmpfileWithContents(t, "echo Hello world!")
```

Comment:
nit: for clarity, use the same pattern that is used for `failingScript` with `successScript2`.

That is, declare it a `string` here and set it in the `"windows"` block below.

Reply:
done